.gitignore

```diff
@@ -7,7 +7,7 @@
 *.swp
 *.swo
 *~
-build*
```

Comment:
Since you are touching this, I have observed that this matches things it shouldn't.

E.g. I had some confusing behaviour when I worked in the test tests/drivers/build_all/.

Is it possible to rename it such that it doesn't match things we want to version control?

build_docs_*/ for instance

Reply:
What if we just ignore `build/` ?